### PR TITLE
Bot rdvgame clarifications #6645

### LINKED ISSUE #6645: Bot rdvgame clarifications
Rename title to be "Spoiler file (Generated in <>)".
Rename seed hash to "Seed Hash for <>: <>"

### PR DESCRIPTION
**Pull request by Logarhythmic**
Fixes #6645 

https://github.com/randovania/randovania/issues/6645

Formatted title as requested, removed permalink

### CODE REVIEW

File: randovania/server/discord/preset_lookup.py [hunk 1]
```diff
@@ -173,15 +173,14 @@ async def reply_for_preset(message: discord.Message, versioned_preset: Versioned
...
     if description.world_count == 1:
         preset = description.get_preset(0)
         embed.description = (
             f"{preset.game.long_name}, with preset {preset.name}.\n"
-            f"Seed Hash: {description.shareable_word_hash}\n"
-            f"Permalink: {description.permalink.as_base64_str}"
+            f"Seed Hash for {preset.game.long_name}: {description.shareable_word_hash}"
         )
         _add_preset_description_to_embed(embed, preset)
     else:
```
Comment (Miepee) [inline randovania/server/discord/preset_lookup.py:186]: you want to change it for multiworld too

File: randovania/server/discord/preset_lookup.py [hunk 2]
```diff
@@ -173,15 +173,14 @@ async def reply_for_preset(message: discord.Message, versioned_preset: Versioned
...
+        title=f"Spoiler file (Generated in {description.randovania_version_text})",
     )
 
     if description.world_count == 1:
         preset = description.get_preset(0)
         embed.description = (
             f"{preset.game.long_name}, with preset {preset.name}.\n"
-            f"Seed Hash: {description.shareable_word_hash}\n"
-            f"Permalink: {description.permalink.as_base64_str}"
+            f"Seed Hash for {preset.game.long_name}: {description.shareable_word_hash}"
```
Comment (Miepee) [inline randovania/server/discord/preset_lookup.py:183]: https://github.com/randovania/randovania/issues/6645#issuecomment-2136793027

> seed hash for \<current version\>: \<current version seed hash\>

Comment (Logarhythmic) [reply randovania/server/discord/preset_lookup.py:183]: fixed it, thought I had merged before the pr

Comment (Miepee) [reply randovania/server/discord/preset_lookup.py:183]: no, this isnt fixed.

**Comment by henriquegemignani** [comment]
It's rather scary this change doesn't need any test change too, I'm rather confident this has test cases. I wonder what's going on

**Comment by Miepee** [comment]
> I'm rather confident this has test cases. I wonder what's going on

Briefly looking into it, I'm pretty sure we only have a look for presets/permalinks being posted, but not actually for rdvgames.

File: randovania/server/discord/preset_lookup.py [hunk 3]
```diff
@@ -173,15 +173,14 @@ async def reply_for_preset(message: discord.Message, versioned_preset: Versioned
 
 async def reply_for_layout_description(message: discord.Message, description: LayoutDescription):
     embed = discord.Embed(
-        title=f"Spoiler file for Randovania {description.randovania_version_text}",
+        title=f"Spoiler file (Generated in {description.randovania_version_text})",
```
Comment (henriquegemignani) [inline randovania/server/discord/preset_lookup.py:176]: I know I suggested this wording, but I'm wondering if "in" is correct. Would "with" be better?

File: randovania/server/discord/preset_lookup.py (hunk 2 above)
Comment (henriquegemignani) [reply randovania/server/discord/preset_lookup.py:183]: ^, this is using the game name again instead of the current version

**Review by henriquegemignani — CHANGES_REQUESTED** [review]

File: randovania/server/discord/preset_lookup.py [hunk 4]
```diff
@@ -173,15 +173,14 @@ async def reply_for_preset(message: discord.Message, versioned_preset: Versioned
...
+        title=f"Spoiler file (Generated with {description.randovania_version_text})",
     )
 
     if description.world_count == 1:
         preset = description.get_preset(0)
         embed.description = (
             f"{preset.game.long_name}, with preset {preset.name}.\n"
-            f"Seed Hash: {description.shareable_word_hash}\n"
-            f"Permalink: {description.permalink.as_base64_str}"
+            f"Seed Hash for {description.randovania_version_text}: {description.shareable_word_hash}"
```
Comment (Miepee) [inline randovania/server/discord/preset_lookup.py:183]: Description.randovania_version_text is wrong here, pretty sure that that's the randovania version that was used to generate the rdvgame. While the word hash here will always show from the current version.
You probably wanna use randovania.VERSION or something like that instead

**Review by henriquegemignani — APPROVED** [review]

**Review by Miepee — DISMISSED** [review]
typod

File: randovania/server/discord/preset_lookup.py [hunk 5]
```diff
@@ -173,15 +174,14 @@ async def reply_for_preset(message: discord.Message, versioned_preset: Versioned
...
+        title=f"Spoiler file (Generated with {description.randovania_version_text})",
     )
 
     if description.world_count == 1:
         preset = description.get_preset(0)
         embed.description = (
             f"{preset.game.long_name}, with preset {preset.name}.\n"
-            f"Seed Hash: {description.shareable_word_hash}\n"
-            f"Permalink: {description.permalink.as_base64_str}"
+            f"Seed Hash for {randovania.version}: {description.shareable_word_hash}"
```
Comment (henriquegemignani) [inline randovania/server/discord/preset_lookup.py:184]: Should be `randovania.VERSION`

**Review by henriquegemignani — CHANGES_REQUESTED** [review]

File: randovania/server/discord/preset_lookup.py [hunk 6]
```diff
@@ -15,6 +15,7 @@
 from discord.ui import Button
 
 import randovania
+import randovania.version
```
[the remark this answers is not in the document]
Comment (Logarhythmic) [reply randovania/server/discord/preset_lookup.py]: removed

File: randovania/server/discord/preset_lookup.py (hunk 5 above)
Comment (Logarhythmic) [reply randovania/server/discord/preset_lookup.py]: fixed

**Review by henriquegemignani — APPROVED** [review]

**Pull request merged**